Tools/Pipeline/Controls/PropertyGridTable.cs

```diff
@@ -176,9 +176,12 @@ private void Drawable_Paint(object sender, PaintEventArgs e)
             else
                 SetCursor(CursorType.Default);
 
+            // On windows craeting a dialog from double click will freeze
+            // the GUI thread until a click occurs so we need to call the
+            // dialog at the end of Paint event so everything gets drawn.
```

Comment:
This sounds crazy...  that never happens under regular C# WinForms or even C++ Win32 applications.

Is this an Eto bug?